scheduler_plugins/slurm/slurm_plugin_infrastructure.cfn.yaml

```diff
@@ -182,7 +182,7 @@ Resources:
                   - logs:CreateLogStream
                   - logs:PutLogEvents
                 Effect: Allow
-                Resource: !Sub "arn:${AWS::Partition}:logs:*:*:*"
+                Resource: !Sub "arn:${AWS::Partition}:logs:${AWS::Region}:${AWS::AccountId}:log-group:/aws/lambda/pcluster-CleanupRoute53Function-*"
```

Comment:
I think the lambda does not have the `pcluster-CleanupRoute53Function-` prefix in its name.

Reply:
Modified to `/aws/lambda/*`